File: AgenciaTurismo/app/src/main/java/com/example/agnciadeturismo/services/MascaraServices.java
package com.example.agnciadeturismo.services;

import android.widget.EditText;

import com.github.rtoshiro.util.format.SimpleMaskFormatter;
import com.github.rtoshiro.util.format.text.MaskTextWatcher;

public class MascaraServices {

    public static void maskFormatter(EditText editText, String mascara) {
        SimpleMaskFormatter mask = new SimpleMaskFormatter(mascara);
        MaskTextWatcher mtw = new MaskTextWatcher(editText, mask);
        editText.addTextChangedListener(mtw);
    }
}

File: AgenciaTurismo/app/src/main/java/com/example/agnciadeturismo/presenter/view/services/MascaraServices.kt
package com.example.agnciadeturismo.presenter.view.services

import android.widget.EditText
import com.github.rtoshiro.util.format.SimpleMaskFormatter
import com.github.rtoshiro.util.format.text.MaskTextWatcher

class MascaraServices {

    companion object{
        fun maskFormatter(editText: EditText, mascara: String?) {
            val mask = SimpleMaskFormatter(mascara)
            val mtw = MaskTextWatcher(editText, mask)
            editText.addTextChangedListener(mtw)
        }
    }
}